dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops

```diff
@@ -633,3 +633,19 @@ ssl.client.handshake.error.CWPKI0824E=CWPKI0824E: SSL HANDSHAKE FAILURE:  Host n
 ssl.client.handshake.error.CWPKI0824E.explanation=An error occurred during the SSL handshake.  Host name verification is enabled and server's identity can not be verified.  The host name used to access the server does not match the server certificate's SubjectDN or Subject Alternative Name information.
 ssl.client.handshake.error.CWPKI0824E.useraction=When host name verification is enable the server's being accessed needs to be setup with a certificate contain the proper information needed to verify the server's identity.
 # -------------------------------------------------------------------------------------------------
+ssl.cacerts.error.CWPKI0825W=CWPKI0825W: The trustCacerts attribute is enabled and there is a failure trying to initialize the JDK default truststore.  The JDK truststore will not be used only the configured truststore will be used.  The [{0}] SSL configuration will be used without the JDK default truststore.  The exception returned is: [{1}].
+ssl.cacerts.error.CWPKI0825W.explanation=There was a problem while trying to setup the JDK''s truststore.  It will not be used for trust.  Only the configured SSL configuration will be used.
+ssl.cacerts.error.CWPKI0825W.useraction=Ensure the JDK's cacerts file exits and is accessible.
+# -------------------------------------------------------------------------------------------------
+ssl.environment.cert.error.CWPKI0826W=CWPKI0826W: An error occurred while trying to get the certificate from the [{0}] environment variable.  The certificate will not be added to the [{1}] truststore.
+ssl.environment.cert.error.CWPKI0826W.explanation=The certificate specified in the environment variable can not be access and will not be included in the truststore.
+ssl.environment.cert.error.CWPKI0826W.useraction=Ensure the environment variable is populated with a base 64 bit environment variable.
+# -------------------------------------------------------------------------------------------------
+ssl.using.cacerts.CWPKI0827I=CWPKI0827I: The [{0}] SSL configuration will use the JDK default truststore in addition to the configured truststore.
+ssl.using.cacerts.CWPKI0827I.explanation=The JDK cacerts file will be used for trust in addition to the truststore configured for the SSL configuration.
+ssl.using.cacerts.CWPKI0827I.useraction=No action is required.
+# -------------------------------------------------------------------------------------------------
+ssl.client.handshake.cacerts.error.CWPKI0828E=CWPKI0828E: The trustCacerts attribute is enabled and there was a failure the establish trust using the JDK default truststore.  The extended error message from the SSL handshake exception is: [{0}].
```

Comment:
Change
"The trustCacerts attribute is enabled and there was a failure the establish trust using the JDK default truststore.  The extended error message from the SSL handshake exception is: [{0}]."
To
"The trustCacerts attribute is enabled and trust was not able to be established with the JDK default truststore. The extended error message from the SSL handshake exception is: [{0}]."